### PR TITLE
className property applied also to FormattedInput

### PR DESCRIPTION
**Pull request by severinoAmmirati**
className property applied also to FormattedInput

### CODE REVIEW

File: src/lib/FormattedInput.tsx [hunk 1]
```diff
@@ -79,11 +80,19 @@ const FormattedInput = <T extends FieldValues>(props: FormattedInputProps<T>) =>
                   }}
                   onFocus={focusHandler}
                   style={style}
+                  className={`form-control ${className}`}
```
Comment (neotrow) [inline src/lib/FormattedInput.tsx:83]: @severinoAmmirati 

Can you please add the `className` as an argument to `classnames` on line 48. Than you can revert the changes here and for the PatternFormat 

Otherwise the invalid class doesn't get rendered.

**Review by neotrow — CHANGES_REQUESTED** [review]
@severinoAmmirati 

Thank you very much for the PR and test 👍

File: src/lib/FormattedInput.tsx (hunk 1 above)
Comment (severinoAmmirati) [reply src/lib/FormattedInput.tsx]: @neotrow Updated as requested

**Review by neotrow — APPROVED** [review]
